DeviceBridgeE2E/src/device-bridge.spec.ts

```diff
@@ -311,3 +310,43 @@ test.serial('Test auth', async t => {
         });
     }
 });
+
+test.serial('Test restart', async t => {
+    // Create subscription to Azure Function
+    const callbackUrl = `${t.context.ctx.callbackUrl}&deviceId=${t.context.device.id}`;
+    await t.context.ctx.deviceBridgAPI.createConnectionStatusSubscription(t, t.context.device.id, callbackUrl)
+    await sleep(3000);
+    // Ensure get works
+    var response = await t.context.ctx.deviceBridgAPI.getConnectionStatusSubscription(t, t.context.device.id);
+    t.is(callbackUrl, response.body.callbackUrl)
+
+    // Ensure connection created event when a sub created
```

Comment:
The test isn't checking the status right after creating the sub as the comment says.